docs/topics/api/reviewers.rst

```diff
@@ -76,9 +75,10 @@ add-on.
        Requires authentication and the current user to have ``Reviews:Admin``
        permission.
 
-.. http:patch::/api/v4/reviewers/addon/(int:addon_id)/flags/
+.. http:patch:: /api/v4/reviewers/addon/(int:addon_id)/flags/
 
     :>json boolean auto_approval_disabled: Boolean indicating whether auto approval are disabled on an add-on or not. When it's ``true``, new versions for this add-on will make it appear in the regular reviewer queues instead of being auto-approved.
     :>json string|null pending_info_request: Deadline date for the pending info request as a string, or ``null``.
     :>json boolean needs_admin_code_review: Boolean indicating whether the add-on needs its code to be reviewed by an admin or not.
     :>json boolean needs_admin_content_review: Boolean indicating whether the add-on needs its content to be reviewed by an admin or not.
+    :>json boolean needs_sensitive_access_review: Boolean indicating whether the add-on needs its content to be reviewed by an admin or not. This is related to the `Addon.require_sensitive_data_access` property.
```

Comment:
These reviews should show up in the regular review queue. All we want is to disable auto approval for add-ons which have access to sensitive data.

I do not think we need to escalate them to admin review directly.

I assume this is just a wording issue here, and not a code logic issue, right?